doc/classes/Node.xml

```diff
@@ -199,7 +199,13 @@
 		<method name="can_process" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the node can receive processing notifications and input callbacks ([constant NOTIFICATION_PROCESS], [method _input], etc) from the [SceneTree] and [Viewport]. The value depends on both the current [member process_mode] and [member SceneTree.paused]. Returns [code]false[/code] if the node is not inside the tree.
+				Returns [code]true[/code] if the node can receive processing notifications and input callbacks ([constant NOTIFICATION_PROCESS], [method _input], etc) from the [SceneTree] and [Viewport]. The returned value depends on [member process_mode]:
+				- When [constant PROCESS_MODE_PAUSABLE], returns [code]true[/code] if the game is not paused, i.e. [member SceneTree.paused] is [code]false[/code];
```

Comment:
Small nitpick (also applies to the other four):
```suggestion
				- If set to [constant PROCESS_MODE_PAUSABLE], returns [code]true[/code] if the game is not paused, i.e. [member SceneTree.paused] is [code]false[/code];
```

Reply:
Fixed.